README.md

```diff
@@ -0,0 +1,50 @@
+<a href="https://www.streamingfast.io/">
+	<img width="100%" src="https://github.com/streamingfast/substreams/blob/develop/docs/assets/substreams-banner.png" alt="StreamingFast Substreams Banner" />
+</a>
+
+# Substreams
```

Comment:
Can you add a readme describing what the sps does. See the other intro sps's for inspiration